### PR TITLE
Change homepage banner for I/O

### PR DESCRIPTION
**Pull request by AaronForinton**

### CODE REVIEW

File: src/site/content/en/index.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ intro:
 featuredCard:
   eyebrow: 'Featured'
   title: 'Watch the progress we made together on the web.'
-  background: ''
-  url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review.'
+  background: '.'
+  url: '.'
```
Comment (rachelandrew) [inline src/site/content/en/index.md:19]:
```suggestion
  url: 'https://io.google/2023/?utm_source=web&utm_medium=embedded_marketing&utm_campaign=hpp_reg_card&utm_content='
```

File: src/site/content/en/index.md [hunk 2]
```diff
@@ -15,10 +15,10 @@ intro:
 featuredCard:
   eyebrow: 'Featured'
   title: 'Watch the progress we made together on the web.'
-  background: ''
-  url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review.'
+  background: '.'
```
Comment (rachelandrew) [inline src/site/content/en/index.md:18]:
```suggestion
  background: ''
```

File: src/site/content/en/index.md [hunk 3]
```diff
@@ -15,10 +15,10 @@ intro:
 featuredCard:
   eyebrow: 'Featured'
   title: 'Watch the progress we made together on the web.'
```
Comment (rachelandrew) [inline src/site/content/en/index.md:17]:
```suggestion
  title: 'Let’s go. It’s Google I/O. Register Now.'
```

**Review by rachelandrew — DISMISSED** [review]
Trying some stuff...

File: src/site/content/en/index.md [hunk 4]
```diff
@@ -14,11 +14,11 @@ intro:
...
 featuredCard:
   eyebrow: 'Featured'
-  title: 'Watch the progress we made together on the web.'
+  title: 'Let’s go. It’s Google I/O. Register Now.'
   background: ''
-  url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review.'
+  url: 'https://io.google/2023/?utm_source=web&utm_medium=embedded_marketing&utm_campaign=hpp_reg_card&utm_content='
+  image: 'https://wd.imgix.net/image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/t6UJBGxTD5XYwq8VgsK5.png'
```
Comment (rachelandrew) [inline src/site/content/en/index.md:20]:
```suggestion
  image: 'image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/t6UJBGxTD5XYwq8VgsK5.png'
```

File: src/site/content/en/index.md [hunk 5]
```diff
@@ -14,11 +14,11 @@ intro:
...
 featuredCard:
   eyebrow: 'Featured'
-  title: 'Watch the progress we made together on the web.'
+  title: 'Let’s go. It’s Google I/O. Register Now.'
   background: ''
-  url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review.'
+  url: 'https://io.google/2023/?utm_source=web&utm_medium=embedded_marketing&utm_campaign=hpp_reg_card&utm_content='
+  image: 'image/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/t6UJBGxTD5XYwq8VgsK5.png'
```
Comment (matthiasrohmer) [inline src/site/content/en/index.md:20]: Sorry, it's confusing. `image` is only considered if `video` is set and then defines a fallback image. For a pure image featured card one is expected to use `background`. Then it looks like this:

image

Not ideal for a11y reasons.

Comment (AaronForinton) [reply src/site/content/en/index.md:20]: Thanks Matthias. We had to do some reconfig to get a video to appear there in the first place so I think I was a little confused.

File: src/site/content/en/index.md [hunk 6]
```diff
@@ -14,11 +14,11 @@ intro:
 
 featuredCard:
   eyebrow: 'Featured'
-  title: 'Watch the progress we made together on the web.'
-  background: ''
-  url: 'https://www.youtube.com/watch?v=W9O4CuSUKb8'
-  video: 'https://web-dev.imgix.net/video/cGQxYFGJrUUaUZyWhyt9yo5gHhs1/lIta9ZJEALbfAzJzmiNl.mp4'
-  alt: 'Chrome icon rolls past a title: 2022 Year in Review.'
+  title: 'Let’s go. It’s Google I/O. Register Now.'
```
Comment (rachelandrew) [inline src/site/content/en/index.md:17]:
```suggestion
  title: 'Let’s go, it’s Google I/O. Register Now.'
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**